rotkehlchen/globaldb/schema.py

```diff
@@ -190,6 +190,8 @@
 INSERT OR IGNORE INTO price_history_source_types(type, seq) VALUES ('C', 3);
 /* XRATESCOM */
 INSERT OR IGNORE INTO price_history_source_types(type, seq) VALUES ('D', 4);
+/* MANUAL_CURRENT */
+INSERT OR IGNORE INTO price_history_source_types(type, seq) VALUES ('E', 5);
```

Comment:
This shoudl also go to the global DB upgrade

Reply:
Oh yes, indeed